app.js

```diff
@@ -31,11 +31,11 @@ class UnifiApp extends Homey.App {
 
         this.log('Freshly initializing com.ubnt.unifi.settings with some defaults')
         this.updateSettings({
-            'host': 'unifi',
-            'port': '8443',
-            'user': 'ubnt',
-            'pass': 'ubnt',
-            'site': 'default'
+            'host': '',
```

Comment:
Why did you remove the defaults?

Reply:
Not really necessary but liked it a bit more while testing